### PR TITLE
Practice01 01 2023

### PR DESCRIPTION
**Pull request by shushkovaira**
version 3 with fixes

### CODE REVIEW

File: src/main/java/libs/ConfigProperties.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import org.aeonbits.owner.Config;
+
+@Config.Sources(value = "file:./src/main/java/resources/config.properties")
+public interface ConfigProperties extends Config {
+        long TIME_FOR_DFFAULT_WAIT();
+        long TIME_FOR_EXPLICIT_WAIT_LOW();
+        long TIME_FOR_EXPLICIT_WAIT_HIGHT();
+
+        String base_url();
```
Comment (tradulenko) [inline src/main/java/libs/ConfigProperties.java:12]: this variables should be deleted

File: src/test/java/baseTest/BaseTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    WebDriver webDriver;
+    Logger logger = Logger.getLogger(getClass());
+    protected LoginPage loginPage;
+
+
+    @Before
+    public void setUp(){
+        logger.info("------"+testName.getMethodName()+" was started-------");
+        initDriver();
+        //      initBaseUrl();
```
Comment (tradulenko) [inline src/test/java/baseTest/BaseTest.java:31]: This commented code should be deleted

Comment (shushkovaira) [reply src/test/java/baseTest/BaseTest.java]: deleted

File: src/main/java/libs/ConfigProperties.java (hunk 1 above)
Comment (shushkovaira) [reply src/main/java/libs/ConfigProperties.java]: deleted

**Comment by tradulenko** [comment]
@KateDrebot could you please review?

**Review by tradulenko — APPROVED** [review]

**Pull request merged**